bot/messaging/events.py

```diff
@@ -284,5 +284,17 @@ def on_assignable_role_remove(self):
         """
         return 'on_assignable_role_remove'
 
+    @property
+    def on_set_deletable(self):
+        """
+        Published when a bot message is needed to be able to be deleted
+        
+        Args:
+            messagesToDelete (List[discord.Message]) Messages to be deleted
+            author (discord.Member) member who called the bot 
+            roles (str) Stores the roles needed to delete the message
+        """
+        return '_on_set_deletable'
```

Comment:
when we switched to the metaclass approach we dont need the underscore prefix anymore

Reply:
Why do most of the classes still have underlines though?

Reply:
do they? they do shoot, i forgot to commit that change -_-. ok nvm then it doesnt matter, i thought id refactored all of them my b

Reply:
I can convert mine and then once its merge I can fix them.